README.md

```diff
@@ -4,7 +4,22 @@ Taskit (said Task-kit or TK for short) is the Masterpoint [Taskfile](https://tas
 
 ## Using Taskit
 
-Taskit is built around the idea of being consumed by downstream repos. It is pulled through our standard installation `Taskfile.yaml`
+Taskit is built around the idea of being consumed by downstream repos. It is pulled through a standard Taskfile that you can find in [`exports/Taskfile.dist.yaml`](./exports/Taskfile.dist.yaml).
+
+The procedure to add it to a project is to do the following:
+
+1. Copy `exports/Taskfile.dist.yaml` to your project.
+2. Add the following lines to your project's `.gitignore`:
```

Comment:
What do you think about adding `.gitignore` modifications to our `task init` flow?